config.json

```diff
@@ -349,6 +349,14 @@
         "prerequisites": [],
         "difficulty": 2
       },
+      {
+        "slug": "robot-simulator",
+        "name": "Robot Simulator",
+        "uuid": "beae3683-46fc-4d4d-bacf-e3bd453d9c17",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 2
```

Comment:
Maybe bump this a bit because it is using some concepts that we don't feature much elsewhere?